__tests__/app.test.js

```diff
@@ -113,8 +113,41 @@ describe('/api/articles', () => {
             expect(body.articles).toBeSortedBy('created_at', {descending: true})
         })
     })
+    test('GET:200 accept a sort_by query, order the response by the provided column and use the default descending order', () => {
```

Comment:
Also worth a test for the default sort of created_at, in descending order. When you don't put any queries on the url, it should still sort according to that!